src/electos/ballotmaker/ballots/contest_data.py

```diff
@@ -0,0 +1,81 @@
+from dataclasses import dataclass, field
+from typing import List
+
+
+@dataclass
+class BallotMeasureData:
+    """Retrieve ballot measure contest data from a dict"""
```

Comment:
Suggestion:
> """Data used to track ballot measure contests (EDF: BallotMeasureContest)"""

- You could leave out the EDF class name, but I believe it helps someone reading it know what type to look up in the spec. There's a lot of overlapping names in the spec and there's always room for confusion 😝 .
- I don't think "from a dict" is necessary: that's a given from how dataclass constructors work.

Reply:
I'm actually thinking that we don't want to refer to the EDF in this code, since your code isolates these classes from the details of the EDF.

Reply:
Just confirming that I agree  💯 we don't want the ballot data models to store the EDF. As we discussed today, the extractor gets the EDF and uses it to populate the ballot data.